pkg/chain_iterator/block_batch_iterator.go

```diff
@@ -19,6 +19,7 @@ import (
 const (
 	DefaultBlocksReadPerEpoch = 1000
 	DefaultRetryInterval      = 12 * time.Second
+	DefaultBlockConfirmations = 6
```

Comment:
```suggestion
	DefaultBlockConfirmations = 0
```
Let's use no confirmations as default.